<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_qgroupbox</class>
 <widget class="QDialog" name="dlg_qgroupbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>691</width>
            <height>555</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="gbGlobalInsert">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="1" column="0">
               <widget class="QGroupBox" name="gbInsert">
                <property name="title">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="pbInsertGroupBox">
                <property name="text">
                 <string>添加包含若干个子部件的QGroupBox</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QPushButton" name="pbInsertToGroupBoxWrong">
                <property name="text">
                 <string>(网上找的，有问题)向QGroupBox中添加若干个子部件</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="pbInsertToGroupBoxRight">
                <property name="text">
                 <string>向QGroupBox中添加若干个子部件</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QGroupBox" name="gbInsertWithLayout">
                <property name="title">
                 <string/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>(ui设计器要求QGroupBox中有东西时才能设置Layout)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QPushButton" name="pbInsertWithLayout">
                <property name="text">
                 <string>(推荐)向QGroupBox的Layout中添加若干个子部件</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>406</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
